Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+
+namespace FindSimilar
+{
+    public class Program
+    {
+        // Create a client.
+        public static string ApiKey = "ENTER YOUR KEY HERE";
+
+        public static IFaceClient Client = new FaceClient(new ApiKeyServiceClientCredentials(ApiKey))
+        {
+            Endpoint = "ENTER YOUR ENDPOINT HERE"
+        };
+
+        public static List<string> TargetImageFileNames =
+            new List<string> { "Family1-Dad1.jpg", "Family1-Daughter1.jpg", "Family1-Mom1.jpg", "Family1-Son1.jpg", "Family2-Lady1.jpg", "Family2-Man1.jpg", "Family3-Lady1.jpg", "Family3-Man1.jpg" };
+
+        public static string SourceImageFileName = "findsimilar.jpg";
+
+        public static void Main(string[] args)
+        {
+            FindSimilar_in_FaceIds().Wait();
+            FindSimilar_in_FaceList().Wait();
+            FindSimilar_in_LargeFaceList().Wait();
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+
+        public static async Task FindSimilar_in_FaceIds()
+        {
```

Comment:
Add `Console.WriteLine("Sample of finding similar faces in face ids.");`
For the other 2 
`Console.WriteLine("Sample of finding similar faces in face list.");`
`Console.WriteLine("Sample of finding similar faces in large face list.");`

Reply:
done